### PR TITLE
feat: #id 21598 refactor RevealAll, MinimizeAll and AutoArrange

### PR DESCRIPTION
**Pull request by meredith-ciq**
[Kanban link](https://chartiq.kanbanize.com/ctrl_board/18/cards/21598/details)

Requires: https://github.com/ChartIQ/finsemble-ui/pull/6

**Description of change**
* Paired with pr to finsemble-ui to wrap these components
* Adds the updated components to toolbar shell.
* Temporary solution for adding toolbar sections - will need to work with team to find a longterm solution.

### CODE REVIEW

File: src-built-in/components/toolbar/src/app.jsx [hunk 1]
```diff
@@ -12,20 +12,27 @@ import React from "react";
...
-			<DragHandle/>
-			<DummyExport/>
-			{/* Drag Handle */}
-			{/* Workspace Management Menu */}
-			{/* App Menu */}
-			{/* Favorites Section */}
-			{/* Minimize All */}
-			{/* Reveal All */}
-			{/* Auto Arrange */}
+			<div className="finsemble-toolbar-section left">
```
Comment (d4hines) [inline src-built-in/components/toolbar/src/app.jsx]: Should we do `div`'s like this or have a specific component like
```jsx
import { ToolbarSection } from "@chartiq/finsemble-ui/components
...
<ToolbarShell>
   <ToolbarSection position=left>
    ...
   </ToolbarSection>
   <ToolbarSection position=center>
    ...
   </ToolbarSection>
   <ToolbarSection position=right>
    ...
   </ToolbarSection>
</ToolbarShell>
```

I can see pro's and con's to both.

Comment (zona-ciq) [reply src-built-in/components/toolbar/src/app.jsx]: I like just doing the div - it gives our client developers more options to customize

File: src-built-in/components/toolbar/src/app.jsx [hunk 2]
```diff
@@ -12,20 +12,27 @@ import React from "react";
...
+				{/* Workspace Management Menu */}
+				{/* App Menu */}
+			</div>
+			<div className="finsemble-toolbar-section center">
+				{/* Favorites Section */}
+			</div>
+			<div className="finsemble-toolbar-section right">
+				<RevealAll />
+				{/* Minimize All */}
+				{/* Auto Arrange */}
```
Comment (zona-ciq) [inline src-built-in/components/toolbar/src/app.jsx]: If we're disabling those two functionalities by commenting this code out, should we import them at the top of this file so people can just use them when those lines are uncommented?

File: src-built-in/components/toolbar/src/app.jsx (hunk 1 above)
Comment (meredith-ciq) [reply src-built-in/components/toolbar/src/app.jsx]: This was meant to be a shortterm solution, we need to decide as a team which we want.

Comment (d4hines) [reply src-built-in/components/toolbar/src/app.jsx]: Terry decided Meredith's current way is best.

**Pull request merged**